src/governance/governance.cpp

```diff
@@ -98,6 +132,34 @@ bool CGovernanceManager::LoadCache(bool load_cache)
     return is_valid;
 }
 
+void CGovernanceManager::RelayMessage(const CGovernanceObject& obj)
+{
+    AssertLockNotHeld(cs_relay);
+    if (!m_mn_sync.IsSynced()) {
+        LogPrint(BCLog::GOBJECT, "%s -- won't relay until fully synced\n", __func__);
+        return;
+    }
+
+    WITH_LOCK(cs_relay, m_relay_invs.emplace_back(MSG_GOVERNANCE_OBJECT, obj.GetHash()));
```

Comment:
Nit: avoid using WITH_LOCK when the scope of the lock will end immediately after anyhow. There's no benefit here to using WITH_LOCK